app/helpers/herbaria_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# helper methods for Herbaria views
+module HerbariaHelper
+  def herbarium_top_users(herbarium)
+    Herbarium.connection.select_rows(%(
+      SELECT u.name, u.login, COUNT(u.id)
+      FROM herbarium_records hr JOIN users u ON u.id = hr.user_id
+      WHERE hr.herbarium_id = #{herbarium.id}
+      GROUP BY u.id ORDER BY COUNT(u.id) DESC LIMIT 5
+    ))
```

Comment:
I just wanted to get this SQL query out of the template.
I like to turn it into an ActiveRecord query, but couldn't figure out how. (And maybe it's not possible.)

Reply:
Nice